backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/SsrService.java

```diff
@@ -0,0 +1,153 @@
+// Acknowledgement to SakaiBorder for providing the template for this code
+// see https://github.com/SakaiBorder/java-vue-ssr/tree/master/src/main/js/src
+
+package org.molgenis.emx2.web;
+
+import static org.molgenis.emx2.graphql.GraphqlTableFieldFactory.createKeyFilter;
+import static org.molgenis.emx2.web.MolgenisWebservice.getSchema;
+import static spark.Spark.get;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.oracle.truffle.js.scriptengine.GraalJSScriptEngine;
+import java.io.*;
+import java.net.MalformedURLException;
+import java.util.Map;
+import javax.script.*;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.HostAccess;
+import org.molgenis.emx2.Filter;
+import org.molgenis.emx2.MolgenisException;
+import org.molgenis.emx2.Schema;
+import org.molgenis.emx2.Table;
+import org.molgenis.emx2.json.JsonUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.resource.AbstractFileResolvingResource;
+import spark.resource.ClassPathResourceHandler;
+
+public class SsrService {
+
+  static Logger logger = LoggerFactory.getLogger(SsrService.class);
+
+  public static String SSR_BASE_PATH = "/public_html/ssr/";
+
+  public static void create() {
+    get("/:schema/view/:table/:filter", SsrService::getRecordForDefaultRoute);
+    get("/:schema/view/:table/:filter/:route", SsrService::getRecordForRoute);
```

Comment:
having global path fiters spread across the code base makes it hard is evaluate if path are being missed due to other paths filters being greedy, suggest to the these in the main server class of in a separate route class